frontend/indigenousapprenticeship/src/css/app.css

```diff
@@ -1 +1,360 @@
-/* nothing here yet */
+.hero__content {
```

Comment:
It seems like a lot of this belongs to just the Home page.  Maybe it should be in a CSS file in that directory?

Reply:
Agreed!

Resolved with [move css to home.css](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2539/commits/bf33d06bf49547ef554322af25d3b824f0f34767)